docker/convert.go

```diff
@@ -53,7 +53,7 @@ func ConvertToAPI(s *Service) (*ConfigWrapper, error) {
 	return &result, nil
 }
 
-func volumes(c *project.ServiceConfig, ctx *Context) map[string]struct{} {
+func volumes(c *project.ServiceConfig, ctx project.Context) map[string]struct{} {
```

Comment:
It's being used here https://github.com/joshwget/libcompose/blob/use-project-context-interface/docker/convert.go#L138